life_dashboard/dashboard/views.py

```
@@ -1,58 +1,126 @@
+    if request.method == "POST":
+        form = AuthenticationForm(request, data=request.POST)
+        if form.is_valid():
+            username = form.cleaned_data.get("username")
+            password = form.cleaned_data.get("password")
+            user = authenticate(username=username, password=password)
+            if user is not None:
+                login(request, user)
+                messages.success(request, "Login successful!")
+                return redirect("dashboard:dashboard")
+            else:
+                messages.error(request, "Invalid username or password.")
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)